officemanager/department/models/models.py

```diff
@@ -1,3 +1,11 @@
 from django.db import models
```

Comment:
All columns of all tables must be required. Currently I can add departments without name or employee without salary, etc.